index.html

```diff
@@ -519,11 +563,102 @@ <h4>Untrusted JavaScript Cloud Conferencing</h4>
            <td>Post Compromise Security (PCS). Protection against
            past or future device compromise.</td>
         </tr>
+        <tr>
+           <td>Nxx</td>
+           <td>Data channels in service workers. - This would allow the page
+            to issue a fetch() which could be resolved in the service worker using P2P 
+            datachannels
+           </td>
+        </tr>
+        <tr>
+         <td>Nxx</td>
+         <td>A group member can encrypt and send copies of the encoded media directly to multiple group members
+            without the intervention of the media server.
+         </td>
+      </tr>
+        <tr><td>note that the requirements from Machine Learning usecase are also required here.</td></tr>
     </tbody>
 </table>
 </section>
 </section>
 </section>
+<section id="decent">
+   <h3>Decentralized internet</h3>
+   <p>New decentralized applications provide P2P services and client-server services for consumption in a browser.</p>
+   <p>The differences in transport layer semantics make it difficult to share code between the 2 modes.</p>
+   <table class="simple">
+       <thead>
+          <tr>
+             <th>Requirement ID</th>
+             <th>Description</th>
+          </tr>
+       </thead>
+       <tbody>
+           <tr>
+              <td>NXX</td>
+              <td>Ability to intercept the fetch API and service it over a P2P link. (one way to do this would be to
+                 support Datachannels in Service Workers - which can already intercept fetch)
+              </td>
+           </tr>
+       </tbody>
+   </table>  
+   <p>Experience: Both |pipe| and [Matrix] have implemented systems of this sort.</p>
+</section>
+<section id="auction">
+   <h3>Low latency P2P broadcast</h3>
+   <p>There are 'broadcast' applications that require low latency realtime media - Distributed auctions or betting for example.</p>
+   <p>The same live video and audio (+data) should be sent to hundreds of recipients</p>
+   <p>WebRTC 1.0 can do this, but it lacks some features that those industries require and have in higher latency streaming technologies.
+   </p>
+   <table class="simple">
+       <thead>
+          <tr>
+             <th>Requirement ID</th>
+             <th>Description</th>
+          </tr>
+       </thead>
+       <tbody>
+           <tr>
+              <td>NXX</td>
+              <td>Predictable auto-play for media elements that works for first time users and is testable.
+              </td>
+           </tr>
+           <tr>
+            <td>NXX</td>
+            <td>Ability to reuse DRM assets streamed over the data channels
+            </td>
+         </tr>
+         <tr>
+            <td>NXX</td>
+            <td>Ability to reuse subtitle assets streamed over the data channels
+            </td>
+         </tr>
+       </tbody>
+   </table>  
+   <p>Experience: Both |pipe| and millicast have built systems of this sort.</p>
+</section>
```

Comment:
While millicast acknowledges that DRM support, Subtitle support, audio channel/track selection (multilanguage), subtitle track selection (multi language), are very frequent request in the field of broadcasting, we are not advocating for any specific implementation. Moreover, we do not support data channel at this stag, so we have not build a system based on the proposed implementation. We believe recent standards around rte payloads for timed text and other approaches might be more suitable to achieve the expected goal.

Reply:
Thanks for the clarification. What I'm trying to indicate is that there is demand for this usecase, which we can't yet satisfy using the webRTC 1.0 spec. Both |pipe| and Millicast have (somewhat) solved these problems based on the current spec, albeit in a less than perfect way. 
Happy to drop the Millicast reference if you like.